addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -170,9 +170,17 @@ def _get_operation_line(self, bom, qty, level):
         operations = []
         total = 0.0
         qty = bom.product_uom_id._compute_quantity(qty, bom.product_tmpl_id.uom_id)
-        for operation in bom.operation_ids:
+        for index, operation in enumerate(bom.operation_ids):
+            time_start = operation.workcenter_id.time_start
+            time_stop = operation.workcenter_id.time_stop
+            if index != 0 and bom.operation_ids[index-1] and bom.operation_ids[index-1].workcenter_id == operation.workcenter_id:
+                time_start = 0
+
+            if index+1 < len(bom.operation_ids.ids) and bom.operation_ids[index+1] and bom.operation_ids[index+1].workcenter_id == operation.workcenter_id:
+                time_stop = 0
+
             operation_cycle = float_round(qty / operation.workcenter_id.capacity, precision_rounding=1, rounding_method='UP')
-            duration_expected = operation_cycle * (operation.time_cycle + (operation.workcenter_id.time_stop + operation.workcenter_id.time_start))
+            duration_expected = operation_cycle * operation.time_cycle + (time_stop + time_start)
```

Comment:
Can you include the efficiency of the work center ?
Like here
https://github.com/odoo/odoo/blob/bb158a724eda39ffc2f51c0bfeb6088b1660451b/addons/mrp/models/mrp_workorder.py#L711